lib/postprocessing/image.js

```diff
@@ -280,6 +310,11 @@ async function applyOutputProperties(img, instructions) {
     if (process.env.SDK_POST_PROCESSING_TEST_MODE) {
         img.set("comment", "Generated by Adobe Asset Compute SDK post-processing.");
     }
+
+    // inject watermarking -compostie should come last 
```

Comment:
Unless it doesn't work otherwise, this should happen above inside `applyOperations()` (line 215 which had the TODO).

`applyOutputProperties()` should be about the final output format - after the image has been rendered/resized/composed.

A clearly defined order will be more important the more features we add here.